build.assets/build-package.sh

```diff
@@ -245,9 +246,22 @@ else
     FILE_LIST="${TAR_PATH}/tsh ${TAR_PATH}/tctl ${TAR_PATH}/teleport ${TAR_PATH}/examples/systemd/teleport.service"
     LINUX_BINARY_FILE_LIST="${TAR_PATH}/tsh ${TAR_PATH}/tctl ${TAR_PATH}/teleport"
     LINUX_SYSTEMD_FILE_LIST="${TAR_PATH}/examples/systemd/teleport.service"
+    EXTRA_DOCKER_OPTIONS=""
+    RPM_SIGN_STANZA=""
     if [[ "${PACKAGE_TYPE}" == "rpm" ]]; then
         OUTPUT_FILENAME="${TAR_PATH}-${TELEPORT_VERSION}-1${OPTIONAL_RUNTIME_SECTION}.${ARCH}.rpm"
         FILE_PERMISSIONS_STANZA="--rpm-user root --rpm-group root --rpm-use-file-permissions "
+        # if we set this environment variable, don't sign RPMs (can be useful for building test RPMs
+        # without having the signing keys)
+        if [ "${UNSIGNED_RPM}" == "true" ]; then
+            echo "RPMs will not be signed as requested"
```

Comment:
to avoid undeclared env vars:
```suggestion
            echo "RPMs will not be signed as requested"
            RPM_SIGN_STANZA=""
```

Reply:
It's already set to the empty variable before the loop on line 250.